client/src/components/FeedbackForm/FeedbackModal.jsx

```diff
@@ -28,8 +28,7 @@ const FeedbackModal = React.memo(({ isOpen, toggle }) => {
           and they will be taken into consideration.
         </p>
         <p>
-          If you need assistance, refer to the{' '}
-          <Link to={`${ROUTES.USER_GUIDE}`}>user guide</Link>, or{' '}
+          If you need assistance,{' '}
```

Comment:
Is this change going to be deployed to all portals, even if they have a user guide? Should this instead be a workbench setting that enables or disables the user guide link?

Reply:
Thought about that too. I was leaning towards the user guide link not being necessary for a portal-specific feedback form submission.

Reply:
Sure I can make it a workbench setting `showUserGuide`. I think I just misunderstood the ticket.